examples/streaming/kafka-hbase-pipeline/src/main/scala/org/apache/gearpump/streaming/examples/pipeline/PipeLineDSL.scala

```diff
@@ -80,12 +81,17 @@ object PipeLineDSL extends App with ArgumentsParser {
     System.setProperty(Constants.GEARPUMP_CUSTOM_CONFIG_FILE, pipeLinePath)
 
     val app = StreamApp("PipeLineDSL", context, appConfig)
-    val producer = app.readFromKafka(kafkaConfig, msg => {
-      val jsonData = msg.msg.asInstanceOf[String]
-      val envelope = read[Envelope](jsonData)
+    val producer = KafkaDSLUtil.createStream[String](app, kafkaConfig, 1, "time-replayable-producer").map{ message =>
```

Comment:
If we have other types of source connectors, how do we extend from KafakDSLUtil? It would be nice to have either implicit conversion to a Source API or a type class mechanism where new source connectors can be added without making them adhere to the DataSource API.